### ISSUE TITLE
ComicInfo.xml "Volume" element seems wrong, possibly should be "Number" instead?

### ISSUE THREAD

**Issue by dropwhile** [manga-py/manga-py#389]
# Bug report
Environment (please complete the following information):

Description | Data
---|---
Manga-py version | 1.29.5

#### To Reproduce:
1. Download a series with a volume name of "67-1" (an example)
2. ComicInfo.xml looks like this in the first few lines:
```
<?xml version="1.0"?>
<ComicInfo xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <Title>blah_blah</Title>
  <Volume>67-1</Volume>
  <PageCount>20</PageCount>
.....
```
3. Trying to use komga to analyze the metadata of the cbz that contains it throws an error:
```
Cannot deserialize value of type `java.lang.Integer` from String "67-1": not a valid Integer value
```
4. Searching for a definition of the comicinfo xml format led me to [this gist](https://gist.github.com/vaemendis/9f3ed374f215532d12bda3e812a130e6).
Seems that Volume should only be an Int. Number is freeform. In this case, I think Number is perhaps more apt since it is a semi-freeform value like "67-1" or "66", depending on the manga site and how they format volume names (manga-py appears to convert "67.1" to "67-1" already so filenames are better).
5. In `download_methods.py`, I changed the following:
```
--- download_methods_old.py	2021-04-27 20:25:39.977518478 -0700
+++ download_methods.py	2021-04-27 20:25:23.633284790 -0700
@@ -159,7 +159,7 @@
     def _book_info_xml(self):
         comic_info = ComicInfo()
 
-        comic_info.volume(self.provider.get_chapter_index())
+        comic_info.number(self.provider.get_chapter_index())
         comic_info.title(self.provider.name)
         comic_info.pages(self.__pages_cache)
         comic_info.page_count(str(len(self.__pages_cache)))
```
Output xml is now:
```
<?xml version="1.0"?>
<ComicInfo xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <Title>blah_blah</Title>
  <Number>67-1</Number>
  <PageCount>20</PageCount>
.....
```
6. With the change komga parses the xml happy, and even seems to get the sorting (individual cbz "book" number) more correct.

**Comment by 1271** [comment]
Thank you for your issue, I'll think about how to fix it

**Comment by dropwhile** [comment]
Another datapoint. I had a fetch result (with the cbz option) that ended up with a volume name of `305-Chapter298`, which resulted in `<Volume>305-Chapter 298</Volume>` in ComicInfo.xml.

**Comment by 1271** [comment]
Do you remember on which site it happened?

**Comment by dropwhile** [comment]
> Do you remember on which site it happened?

`https://bato [.] to/series/83648/boku-no-hero-academia-official` (url munged a bit to not link directly)

**Comment by 1271** [comment]
Thanks!
I'm a little busy right now, but next week there will be some free time and I will try to fix it.

**Comment by dropwhile** [comment]
No worries. In the meantime I've got a workaround locally that seems to be doing the job for now.
❤️

**Comment by dropwhile** [comment]
fixed by f849d2a3ec6998e08b2b17637190d90fbfea7339
